src/xenia/kernel/xam/content_manager.cc

```diff
@@ -259,6 +261,26 @@ std::filesystem::path ContentManager::ResolveGameUserContentPath() {
   return root_path_ / title_id / kGameUserContentDirName / user_name;
 }
 
+void ContentManager::CloseOpenedFilesFromContent(
```

Comment:
This needs a `TODO: clean this up eventually` or so. Iterating all open file handles to do this seems like a bad idea in the long run.